src/H5FDonion.c

```diff
@@ -351,6 +354,65 @@ H5Pset_fapl_onion(hid_t fapl_id, const H5FD_onion_fapl_info_t *fa)
     FUNC_LEAVE_API(ret_value)
 } /* end H5Pset_fapl_onion() */
 
+hssize_t
+H5FDget_onion_revision_count(const char *filename, hid_t fapl_id)
+{
+    H5P_genplist_t *plist        = NULL;
+    ssize_t         ret_value    = 0;
+    H5FD_t *        file_drv_ptr = NULL;
+    FUNC_ENTER_API(FAIL)
+
```

Comment:
Should check filename for NULL, zero length on entry

Reply:
Done.